components/FlexibleHeader/README.md

```diff
@@ -261,6 +261,14 @@ self.headerViewController.headerView.trackingScrollView = scrollView;
 
 `scrollView` might be a table view, collection view, or a plain UIScrollView.
 
+#### iOS 13 Collection Considerations
+
+iOS 13 changed the behavior of the `contentInset` of a collection view by triggering a layout.
+This may affect your app if you have not yet registered cells for reuse yet. Our recomendation is
+to use view controller composition by making your collection view a child view controller. If this
+is not possible then ensuring your order of opperations has cells registered before setting the
```

Comment:
Nit: Suggested rewrite, "...then ensure the correct order of operations by registering cell reuse identifiers before setting the Flexible Header's `trackingScrollView`."

Reply:
Done.